statsmodels/tsa/tests/test_stattools.py

```diff
@@ -349,6 +349,13 @@ def test_coint():
             r1 = res1[i][2]
             assert_allclose(r1, r2, rtol=0, atol=6e-7)
 
+    # use default autolag #4490
+    res1_0 = coint(y[:, 0], y[:, 1], trend='ct', maxlag=4)
+    assert_allclose(res1_0[2], res_egranger['ct'][0][1:], rtol=0, atol=6e-7)
+    # the following is just a regression test
+    assert_allclose(res1_0[:2], [-13.992946638547112, 2.270898990540678e-27],
+                    rtol=1e-10, atol=1e-27)
```

Comment:
isnt `atol` of `1e-27` going to allow `res1_0[1]` to be off by almost half and still pass?

Reply:
That's intentional. 
I have no idea how large the noise is and how large the noise effect is on the p-values. Also, whether the pvalue is 1e-16 or 1e-28 is not really relevant, I just wanted to check that it is in the right neigborhood without getting noisy failures across machines or LAPACK versions.
The assert on the test statistic is reasonably strict at rtol=1e-10.

Reply:
As long as its intentional that's fine by me, mostly just-checking.

Non-determinism in arithmetic ops still strikes me as super-weird...